src/Components/form.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+function Form() {
+  return (
+    <div className="add-form">
+      <form method="POST" className="form1">
+        <lebel>ADD NEW BOOK</lebel>
+        <br />
+        <input type="text" placeholder="Book title" />
+        <br />
+        <input type="text" placeholder="Author" />
+        <br />
+        <button type="submit">Check status</button>
```

Comment:
- [ ] Since this form is about adding books, let's change the `Check status` text to `Add a book` to improve the user experience.